code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -74,7 +74,7 @@
 		if(health <= HEALTH_THRESHOLD_DEAD)
 			death()
 			return
-		else if(stat == UNCONSCIOUS)
+		else if(stat >= UNCONSCIOUS)
```

Comment:
not really something you'd have to fix for this PR but this seems like strange behavior, when would a silicon be unconscious or soft crit?

Reply:
I addressed this in the body of the PR:
`Cleaned some code and added some not-entirely-necessary checks for mobs that cannot yet be in hard crit, but should be helpful should anyone want to add support for that further down the line.`